src/js/mep-player.js

```diff
@@ -1,3 +1,14 @@
+function isAfter(node, otherNode) {
```

Comment:
yeah, i like sourceNode and targetNode.  plus a little jsdoc is a nice touch.  tacking a function onto jquery for just this use seems a bit heavy.  don't know if it would discourage upstream from accepting it or not.